### PR TITLE
Removed empty checkAndCreateTopic method from wfm

### PR DESCRIPTION
**Pull request by timofei-durakov**
method is not used and doesn't contain any logic
This patch remove this method

### CODE REVIEW

File: services/wfm/src/main/java/org/openkilda/wfm/topology/islstats/IslStatsTopology.java [hunk 1]
```diff
@@ -53,14 +52,16 @@ public StormTopology createTopology() {
...
-        checkAndCreateTopic(openTsdbTopic);
         KafkaBolt openTsdbBolt = createKafkaBolt(openTsdbTopic);
         builder.setBolt(ISL_STATS_OTSDB_BOLT_ID, openTsdbBolt, topologyConfig.getParallelism())
                 .shuffleGrouping(ISL_STATS_BOLT_ID);
 
         return builder.createTopology();
     }
 
+    /**
+     * main.
```
Comment (nikitamarchenko) [inline services/wfm/src/main/java/org/openkilda/wfm/topology/islstats/IslStatsTopology.java]: Is that workaround for checkstyle?

File: services/wfm/src/main/java/org/openkilda/wfm/topology/portstate/PortStateTopology.java [hunk 2]
```diff
@@ -101,14 +98,16 @@ public StormTopology createTopology() throws NameCollisionException {
...
-        checkAndCreateTopic(speakerTopic);
         KafkaBolt speakerBolt = createKafkaBolt(speakerTopic);
         builder.setBolt(SPEAKER_KAFKA_BOLT_NAME, speakerBolt, topologyConfig.getParallelism())
                 .shuffleGrouping(SWITCH_PORTS_SPOUT_NAME);
 
         return builder.createTopology();
     }
 
+    /**
+     * main.
```
Comment (sergii-iakovenko) [inline services/wfm/src/main/java/org/openkilda/wfm/topology/portstate/PortStateTopology.java]: I'd recommend the following text: Entry point for local run of the topology.

File: services/wfm/src/main/java/org/openkilda/wfm/topology/islstats/IslStatsTopology.java (hunk 1 above)
Comment (sergii-iakovenko) [reply services/wfm/src/main/java/org/openkilda/wfm/topology/islstats/IslStatsTopology.java]: I'd recommend the following text: Entry point for local run of the topology.

File: services/wfm/src/main/java/org/openkilda/simulator/SimulatorTopology.java [hunk 3]
```diff
@@ -86,13 +84,15 @@ public StormTopology createTopology() {
...
         // TODO(dbogun): check is it must be output topic
-        checkAndCreateTopic(inputTopic);
         builder.setBolt(KAFKA_BOLT, createKafkaBolt(inputTopic), parallelism)
                 .shuffleGrouping(SWITCH_BOLT, KAFKA_BOLT_STREAM);
 
         return builder.createTopology();
     }
 
+    /**
+     * main.
```
Comment (sergii-iakovenko) [inline services/wfm/src/main/java/org/openkilda/simulator/SimulatorTopology.java]: I'd recommend the following text: Entry point for local run of the topology.

**Pull request merged**